src/components/screens/monitor/delegates/latestVotes/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Table from '../../../../toolbox/table';
+import VotesRow from './voteRow';
```

Comment:
This component is defined as `VoteRow` but in this file referred to as `VotesRow`. It can be changed here
```suggestion
import VoteRow from './voteRow';
```

Or in the other file, but both should use the same name